### ISSUE TITLE
config option <cursor:no_hardware_cursors> does not exist

### ISSUE THREAD

**Issue by masalskiyivan** [hyprwm/Hyprland#7148]
### Regression?

No

### System Info and Version

<details> <summary>System/Version info</summary>

```sh
Hyprland, built from branch  at commit fe7b748eb668136dd0558b7c8279bfcd7ab4d759  (props: bump version to 0.39.1).
Date: Tue Apr 16 16:01:03 2024
Tag: v0.39.1, commits: 4460

flags: (if any)


System Information:
System name: Linux
Node name: desktop-e5vkmhh
Release: 6.9.12-200.fc40.x86_64
Version: #1 SMP PREEMPT_DYNAMIC Sat Jul 27 15:56:15 UTC 2024


GPU information: 
08:00.0 VGA compatible controller [0300]: NVIDIA Corporation GP107 [GeForce GTX 1050] [10de:1c81] (rev a1) (prog-if 00 [VGA controller])


os-release: NAME="Fedora Linux"
VERSION="40 (Workstation Edition)"
ID=fedora
VERSION_ID=40
VERSION_CODENAME=""
PLATFORM_ID="platform:f40"
PRETTY_NAME="Fedora Linux 40 (Workstation Edition)"
ANSI_COLOR="0;38;2;60;110;180"
LOGO=fedora-logo-icon
CPE_NAME="cpe:/o:fedoraproject:fedora:40"
DEFAULT_HOSTNAME="fedora"
HOME_URL="https://fedoraproject.org/"
DOCUMENTATION_URL="https://docs.fedoraproject.org/en-US/fedora/f40/system-administrators-guide/"
SUPPORT_URL="https://ask.fedoraproject.org/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"
REDHAT_BUGZILLA_PRODUCT="Fedora"
REDHAT_BUGZILLA_PRODUCT_VERSION=40
REDHAT_SUPPORT_PRODUCT="Fedora"
REDHAT_SUPPORT_PRODUCT_VERSION=40
SUPPORT_END=2025-05-13
VARIANT="Workstation Edition"
VARIANT_ID=workstation


plugins:

```

</details>

### Description

I pasted NVIDIA variables from [wiki](https://wiki.hyprland.org/Nvidia/) to Hyprland config.
After this, an error is shown at the top of the screen:
```
cursor:no_hardware_cursors does not exist
cursor:allow_dumb_copy does not exist
```
Rebooting doesn't fix it.

### How to reproduce

Just install hyprland 
`sudo dnf install hyprland`

### Crash reports, logs, images, videos

![tzM6dz_7Vls](https://github.com/user-attachments/assets/3e4cfa62-665c-4879-9770-16d006eef6f9)

**Comment by skystar-p** [comment]
Your Hyprland version is too old. That options available starting from v0.41.0.

**Comment by ikalco** [comment]
for some reason v0.39.1 wiki is missing but this should also work
https://wiki.hyprland.org/0.39.0/Nvidia

**Comment by masalskiyivan** [comment]
> Your Hyprland version is too old. That option is available starting from v0.41.0.

LOL why
How to update ? Update & upgrade don't help

**Comment by masalskiyivan** [comment]
> for some reason v0.39.1 wiki is missing but this should also work https://wiki.hyprland.org/0.39.0/Nvidia

Works! thx)
